### ISSUE TITLE
Select mode returning undefined when selecting the first option

### ISSUE THREAD

**Issue by Pacheco95** [terkelg/prompts#152]
### Describe the bug
Selecting the first option without using the arrow keys is always returning `undefined`

### To Reproduce

```javascript
async function askAndReturnAnswers() {
  const questions = [
    {
      type: 'select',
      name: 'searchTerm',
      message: 'Choose one search term:',
      choices: await trends.searchHotTrends(count=9),
      validate: value => typeof value === 'string' ? value.trim() !== '' : false,
    },
    {
      type: 'select',
      name: 'prefix',
      message: 'Choose one prefix:',
      choices: ['Who is', 'What is', 'The history of'],
      validate: value => typeof value === 'string' ? value.trim() !== '' : false,
    }
  ];

  return new Promise(async (resolve, reject) => {
    const promptOptions = {
      onCancel: () => reject(new Error('The user has stopped answering'))
    }
    const response = await prompts(questions, promptOptions)
    resolve(response)
  });
}

async function start() {
  const content = await askAndReturnAnswers()
  console.log(content);
}

start();
```

### Expected behavior
The returned value to return the first value in selection

### System

- OS: Linux Mint 19.1 (Tessa)
- Terminal: Bash
- Node version: v8.10.0
![out](https://user-images.githubusercontent.com/12385467/54161620-7ba4e080-4431-11e9-961e-c219a44698b6.gif)

**Comment by lumio** [comment]
This is indeed confusing behaviour. The problem is, that `prompts` expects an array of choice-objects, like this:

```js
[
  { title: 'Who is', value: 0 },
  { title: 'What is', value: 1 },
  { title: 'The history of', value: 2 },
]
```

I agree however to allow a simple array of strings that generates that exact array of choice objects instead.
Funny enough: It already works with node 10+

**Comment by lumio** [comment]
Fixed with #151 in v2.0.4. Thanks for the raised issue!
